stable/datadog/values.yaml

```diff
@@ -293,18 +259,210 @@ datadog:
       cpu: 200m
       memory: 256Mi
 
+## @param clusterAgent - object - required
+## This is the Datadog Cluster Agent implementation that handles cluster-wide
+## metrics more cleanly, separates concerns for better rbac, and implements
+## the external metrics API so you can autoscale HPAs based on datadog metrics
+## ref: 
+#
+clusterAgent:
+  containerName: cluster-agent
```

Comment:
These are not documented (ditto for replicas), is it because that would be redundant with the previous similar options for the agent contaner?

Reply:
yup